test/tests/parallel/set1/spam_virus/rspamd_full.bats

```diff
@@ -279,9 +286,9 @@ function teardown_file() { _default_teardown ; }
   _nc_wrapper 'nc/rspamd_imap_move_to_inbox.txt' '0.0.0.0 143'
   sleep 1 # wait for the transaction to finish
 
-  _run_in_container cat /var/log/mail/mail.log
-  assert_success
-  assert_output --partial 'imapsieve: Matched static mailbox rule [2]'
+  _service_log_should_contain_string 'mail' 'imapsieve: Matched static mailbox rule \[2\]'
+
+  _filter_service_log 'mail' '.'
```

Comment:
This usage relying on regex, along with some other usage suggests you could have a method that just returns the log, or defaults the 2nd parameter to `.` if it's not provided? 🤔

Reply:
> This usage relying on regex, along with some other usage suggests you could have a method that just returns the log

I explicitly opted against such a method because

1. we do not need it in many places really, filtering is always nicer;
2. it is probably a bit slower than filtering (with `grep`) (I haven't verified this, though);
3. I want future test writers to use filters instead of just `cat`ing the whole log, which may seem easy, but it is also error-prone.

> or defaults the 2nd parameter to `.` if it's not provided? 🤔

This is, IMO, a confusing default and hence, one I don't like. Especially with upcoming changes of not using a regex by default but a static string, we need to carefully think what to do here.

Reply:
Done in aaeea8af8100fa52e5087abe87b007726e7826de